scripts/setup.py

```diff
@@ -0,0 +1,13 @@
+from setuptools import setup
+
+setup(
+    name='securedrop',
+    version='1.0',
+    packages=['securedrop', 'securedrop.tests'],
+    url='https://github.com/aokellermann/securedrop.git',
```

Comment:
Remove trailing `.git`. It should link to project homepage, not clone link.

Reply:
Changed